package.json

```diff
@@ -12,6 +12,7 @@
   "dependencies": {
     "@google-cloud/compute": "^2.5.0",
```

Comment:
Carets

Reply:
Whole that plugin requires refactoring.
Fixed.